package.json

```diff
@@ -63,6 +63,7 @@
     "bootstrap-sass": "^3.3.7",
     "classnames": "^2.2.5",
     "lodash": "^4.17.4",
+    "npm": "^6.1.0",
```

Comment:
This project runs on `yarn` so you don't need `NPM`

I would delete your `node_modules` directory and `package-lock.json`, then run `yarn install` to set yourself back up with `yarn`. Then check it all still works

Reply:
You can uninstall `npm` now